lisp/ess-inf.el

```diff
@@ -1625,18 +1624,27 @@ region this is."
     ;; External applications might call ess-eval-* functions; make it
     ;; easier for them
     (ess-setq-vars-local (symbol-value (ess-get-process-variable 'ess-local-customize-alist))))
-
+  (if (and (bound-and-true-p rectangle-mark-mode)
+           (fboundp 'extract-rectangle-bounds))
```

Comment:
I guess this should be removed with emacs 25 right?

Reply:
Actually now that I think about it we can get rid of it now since we're already checking rectangle-mark-mode. Just need to tell the byte compiler it's OK with a `declare-function`. Will update and push soon.

Reply:
Argh, turns out we can't because rectangle-mark-mode can be bound in Emacs 24.5 but extract-rectangle-bounds wasn't added until Emacs 25.1. I added a TODO about this.